lib/utils/sleep.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function (ms) {
```

Comment:
a module for this is overkill, just duplicate the function in the two modules that use it

Reply:
> 

Done.